### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_Keyboard:D drive is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6032]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name: Voice Over
MAS Reference:MAS2.1.1
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab from Home page->Click on new job button->Select steps tab.
4.Go to the "New step" button hit enter/spacebar.
5.Navigate to "Advanced' button and navigate with Tab till open profile button.
6.Hit space bar/enter to navigate to next page.
7.Hit tab to navigated the interactive item on the page.
**Actual:**
when we navigate the drive through keyboard by hitting tab keyboard focus is not going on D drive.
**Expected:**
keyboard focus should go on D drive when we navigate it through tab.
**User Impact:**
User with motor disabilities will find it difficult to navigate.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
**MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
**Attachment for Reference:**
[6032 _Accessibility]A11y_AzureDataStudio_NewJobSteps_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3290743/6032._Accessibility.A11y_AzureDataStudio_NewJobSteps_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by qifahs** [comment]
It's not possible to navigate through the tree structure using the keyboard.
